server/apple_notification_server.go

```diff
@@ -94,6 +98,10 @@ func (me *AppleNotificationServer) SendNotification(msg *PushNotification) PushR
 		data.Custom("sender_id", msg.SenderId)
 	}
 
+	if len(msg.SenderName) > 0 {
```

Comment:
Looks not related stuff, but I don't see any problem to include this new data as part of the PR if that is on purpose :)